contrib.php

```diff
@@ -39,10 +38,11 @@ function multidimsort($array_in, $column)
     reset($multiarray);
 
     // traverse new array of index values and add the corresponding element of the input array to the correct position in the output array
-    while (list ($key, $val) = each ($multiarray))
+    while (current($multiarray))
```

Comment:
Again very complex- why not use a simple foreach loop instead?

Reply:
sry my failure you need the key of the array-Element and i has no idea how to get it is this better?

Reply:
You can do this:

    Foreach($foo as $key => $val) 

and ignore $val if you dont need it. Sorry, I could have been clearer here.

Reply:
thanks changed